contracts/src/v0.8/automation/dev/chains/ArbitrumModule.sol

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.19;
```

Comment:
I think the modules should be fixed to a specific version

Reply:
done